app/controllers/infra_networking_controller.rb

```diff
@@ -103,14 +103,6 @@ def explorer
       return
```

Comment:
@h-kataria my guess is that tis `request.xml_http_request?` above is also unnecessary, but we might have some kind of magic in the area, can you please help finding it out?